README.md

```diff
@@ -13,32 +13,60 @@ Describe your example and specify what it is demonstrating. Consider the
 following questions:
 
 * Is it pedagogical or a usable application?
-* Does it belong to a particular domain such as material simulation or logistics? 
-* What level of Ocean proficiency does it target (beginner, advanced, pro)? 
+* Does it belong to a particular domain such as material simulation or logistics?
+* What level of Ocean proficiency does it target (beginner, advanced, professional)?
 
 A clear description allows us to properly categorize your example.
 
 Images are encouraged. If your example produces a visualization, consider
 displaying it here.
 
-![D-Wave Logo](dwave_logo.png)
+![D-Wave Logo](static/dwave_logo.svg)
 
-## Usage
+## Installation
```

Comment:
We're switching between instructions to the example contributor and boilerplate text for example users without distinguishing the audiences. Should we use comments or something before sections that the example contributor should keep as is?